assets/tests/components/detours/detourMap.test.tsx

```diff
@@ -20,6 +20,7 @@ const DetourMapWithDefaults = (
     endPoint={undefined}
     waypoints={[]}
     undoDisabled={false}
+    originalShapeClickable={true}
```

Comment:
**suggestion:** is it possible to add a couple of tests for what happens when this prop is enabled / disabled?

Reply:
[How about these?](https://github.com/mbta/skate/pull/2434/commits/e205afd809a220afdd339af1d747bc7ab8a0c30b)